### PR TITLE
fix(editor): Handle disabled nodes in schema view

### PR DESCRIPTION
**Pull request by elsmr**
## Summary

Schema view should still render a list of nodes when the previous node is disabled.
Disabled nodes should be marked Deactivated and have a description when expanded:

image

## Related Linear tickets, Github issues, and Community forum posts

https://linear.app/n8n/issue/NODE-1496/schema-view-handle-disabled-nodes

## Review / Merge checklist

- [ ] PR title and summary are descriptive. ([conventions](../blob/master/.github/pull_request_title_conventions.md)) 
- [ ] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [ ] Tests included. 
- [ ] PR Labeled with `release/backport` (if the PR is an urgent fix that needs to be backported)

### CODE REVIEW

**Review by ShireenMissi — DISMISSED** [review]
LGTM 👍

File: packages/editor-ui/src/components/RunData.vue [hunk 1]
```diff
@@ -253,11 +253,14 @@
...
 			</div>
 
-			<div v-else-if="!hasNodeRun" :class="$style.center">
+			<div v-else-if="!hasNodeRun && !(isInputSchemaView && node?.disabled)" :class="$style.center">
 				<slot name="node-not-run"></slot>
 			</div>
 
-			<div v-else-if="paneType === 'input' && node?.disabled" :class="$style.center">
+			<div
+				v-else-if="paneType === 'input' && node?.disabled && !isInputSchemaView"
```
Comment (ShireenMissi) [inline packages/editor-ui/src/components/RunData.vue:261]: Nitpick : could you please rearrange the order of these conditions 
```suggestion
				v-else-if="paneType === 'input' && !isInputSchemaView && node?.disabled"
```

**Review by ShireenMissi — COMMENTED** [review]
Left a minor comment

**Review by ShireenMissi — APPROVED** [review]
LGTM 🎉

**Comment by janober** [comment]
Got released with [`n8n@1.51.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.51.0)

**Pull request merged**